x-pack/filebeat/module/aws/s3_server_access_log/config/config.yml

```diff
@@ -0,0 +1,16 @@
+{{ if eq .input "s3" }}
+
+type: s3
+queue_url: {{ .queue_url }}
+credential_profile_name: {{ .credential_profile_name }}
+
+{{ else if eq .input "file" }}
+
+type: log
+paths:
+  {{ range $i, $path := .paths }}
+- {{$path}}
+  {{ end }}
+exclude_files: [".gz$"]
+
+{{ end }}
```

Comment:
Instead of using elsifs, I'd separate this configuration in two files as we do for example in the [haproxy log metricset](https://github.com/elastic/beats/tree/v7.3.0/filebeat/module/haproxy/log).